impress-org/givewp-next-gen
# Feature: classic template

Resolves: #42



## Description



This introduces the first template for our next-gen forms: _Classic Template_.  This is not a complete representation of our existing classic template because our form's functionality is not necessarily the same yet.  As we carry over more features & blocks like _add-on_ functionality, this template will be updated.  This PR lays more of the ground-work for building templates and gives us an _initial_ template to continue building out.

## Affects



- Our Template Api & base fields structure
- Next-Gen donation form block defaults to classic template for now
- Block render updated to match form builder persistence updates (different names for things)

## Visuals



<img width="819" alt="Screen Shot 2022-07-25 at 11 31 02 AM" src="https://user-images.githubusercontent.com/10138447/180817613-eb0f4ae4-979f-4bd5-bbd7-8ecd1220e84d.png">

<img width="819" alt="Screen Shot 2022-07-25 at 11 33 48 AM" src="https://user-images.githubusercontent.com/10138447/180817646-2bff3b58-c2c9-470b-9729-f899a3eceed1.png">


## Testing Instructions



- Create a new form using the next-gen form builder
- Add a donation form block to a page
- Make sure you have either Next Gen Test Gateway or Next Gen Stripe Gateway enabled
- Viewing the form should be visually similar to our good ol' Classic Template
- Test out the Form!

## Pre-review Checklist
